test/base_factory.js

```diff
@@ -0,0 +1,16 @@
+const { assertInvalidOpcode } = require('./helpers/assertThrow')
+const BaseFactory = artifacts.require('BaseFactory')
+
+contract('BaseFactory', accounts => {
```

Comment:
Can we add some tests to check that apps and permissions have been set up and deployed correctly?